packages/main/src/components/add-prefix-modal/utils/Api.jsx

```diff
@@ -0,0 +1,12 @@
+import { BASE_API_URL } from "@zuri/utilities";
+import axios from "axios";
+
+const endpoint = BASE_API_URL;
```

Comment:
why the rename if you do not plan to use it?

Reply:
I removed it since its not been used in the component I imported from old to new src